src/components/menus/AppMenu.svelte

```diff
@@ -88,6 +91,15 @@
       <InfoCircleIcon />
       About
     </MenuItem>
+    <!-- Render user display as a button to intercept click events and prevent menu closure on click -->
+    <button
+      class="st-button tertiary app-menu--user"
+      on:click|capture={evt => {
+        evt.stopPropagation();
+      }}
+    >
+      Logged in as&nbsp;<span class="st-typography-medium">{user?.id || 'Unknown'}</span>
```

Comment:
is the `&nbsp;` necessary to enforce the spacing?

Reply:
Refactored this a bit - was necessary since the button is using flex but nested it all under a span so now no longer requires that explicit space.